client/components/CheckoutCalendar/CheckoutCalendar.js

```diff
@@ -1,22 +1,37 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import axios from 'axios';
+import { numberOfGuests, totalReviewCount, averageReviewRatings } from '../../../sampleData/sampleData';
+
 
 class CheckoutCalendar extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      allowedGuests: numberOfGuests.numberOfGuests,
```

Comment:
why are you storing numberOfGuests in state? will numberOfGuests update as the user interacts with the checkout component? or will it remain the same based on the current property?

Reply:
good call out i'll declare it as a const instead since it'll never need to be changed thanks